src/remoteplayer.h

```diff
@@ -32,6 +32,16 @@ enum RemotePlayerChatResult
 	RPLAYER_CHATRESULT_KICK,
 };
 
+struct CloudSettings
```

Comment:
We often use abbreviations like 'def' and 'gen' in file names, and we have a file called gameparams.h https://github.com/minetest/minetest/blob/master/src/gameparams.h
So please can the file be called cloudparams.h? And sorry for my change of request.